core/domain/wipeout_service_test.py

```diff
@@ -4613,3 +4613,185 @@ def test_verify_user_delete_when_external_auth_associations_are_not_deleted(
                 wipeout_service.get_pending_deletion_request(self.user_1_id))
 
         self.assertFalse(wipeout_service.verify_user_deleted(self.user_1_id))
+
+
+class WipeoutServiceDeleteBlogPostsModelsTests(test_utils.GenericTestBase):
+    """Provides testing of the deletion part of wipeout service."""
+
+    BLOG_1_ID = 'blog_1_id'
+    USER_1_EMAIL = 'some@email.com'
+    USER_1_USERNAME = 'username1'
+    USER_2_EMAIL = 'some-other@email.com'
+    USER_2_USERNAME = 'username2'
+    NUMBER_OF_MODELS = 150
+    NONEXISTENT_USER_ID = 'id_x'
+    CONTENT = 'Dummy Content'
+    SUMMARY = 'Dummy Content'
+    TITLE = 'Dummy Title'
+    TAGS = 'tag1 tag2 tag3'
+    THUMBNAIL = 'xyzabc'
+
+    def setUp(self):
+        super(WipeoutServiceDeleteBlogPostsModelsTests, self).setUp()
+        self.signup(self.USER_1_EMAIL, self.USER_1_USERNAME)
+        self.signup(self.USER_2_EMAIL, self.USER_2_USERNAME)
+        self.user_1_id = self.get_user_id_from_email(self.USER_1_EMAIL)
+        self.user_2_id = self.get_user_id_from_email(self.USER_2_EMAIL)
+        self.blog_post_model = blog_models.BlogPostModel(
+            id=self.BLOG_1_ID,
+            author_id=self.user_1_id,
+            content=self.CONTENT,
+            title=self.TITLE,
+            last_updated=datetime.datetime.utcnow(),
+            published_on=datetime.datetime.utcnow(),
+            url_fragment='sample-url-fragment',
+            tags=self.TAGS,
+            thumbnail_filename=self.THUMBNAIL
+        )
+        self.blog_post_model.update_timestamps()
+        self.blog_post_model.put()
+        self.blog_post_summary_model = blog_models.BlogPostSummaryModel(
+            id=self.BLOG_1_ID,
+            author_id=self.user_1_id,
+            summary=self.SUMMARY,
+            title=self.TITLE,
+            last_updated=datetime.datetime.utcnow(),
+            published_on=datetime.datetime.utcnow(),
+            url_fragment='sample-url-fragment',
+            tags=self.TAGS,
+            thumbnail_filename=self.THUMBNAIL
+        )
+        self.blog_post_summary_model.update_timestamps()
+        self.blog_post_summary_model.put()
+        wipeout_service.pre_delete_user(self.user_1_id)
+        wipeout_service.pre_delete_user(self.user_2_id)
+        self.process_and_flush_pending_tasks()
+
+    def test_one_blog_model_is_pseudonymized(self):
+        wipeout_service.delete_user(
+            wipeout_service.get_pending_deletion_request(self.user_1_id))
+
+        # Verify user is pseudonymized.
+        blog_mappings = (
```

Comment:
nit pick: change to `pseudonymizable_user_id_mapping` to help readability

Reply:
Done